app/code/community/Bolt/Boltpay/Model/Observer.php

```diff
@@ -95,13 +95,14 @@ public function verifyOrderContents($observer)
         if (strtolower($method) == Bolt_Boltpay_Model_Payment::METHOD_CODE) {
 
             $reference = $payment->getAdditionalInformation('bolt_reference');
+            $magento_total = Mage::getStoreConfig('tax/sales_display/grandtotal', $order->getStoreId()) ? (int)($order->getGrandTotal()*100) : (int)(($order->getGrandTotal()+$order->getTaxAmount())*100);
```

Comment:
In my mind the transaction amount would always include tax, doesn't it?

Reply:
To Daniel's point, I don't think I've ever seen a grand total that didn't include tax, but if you have I'd be interested in knowing what caused it. Also, casing should probably be camel case.

Reply:
@dumega In my mind too.
@johnmccombs1 I never saw it before until I ran into a case where it made it clear.  The case is in the configuration listed above.

+1 for camelCase.  The change will be applied.